### DISCUSSION TITLE
Szablony ustawień gry

### DISCUSSION THREAD

**Discussion by pmic0** [pmic0/terraforming-mars#3] [Ideas]
Możliwość przygotowania szablonów i wgrania ich na serwer.
Wtedy zamiast ładować z pliku będzie można je też wybrać z listy rozwijanej.
Wstępnie wgrywane szablony będą przez administratora, jako pliki .json w gicie.
Pewnie oprócz nazwy przydałby się też jakiś opis szablonu. Może dodatkowe pole na stronie 
zakładania gry

**Comment by Ziouo** [comment]
I would give this thread top priority
